posthog/api/test/test_organization_feature_flag.py

```diff
@@ -140,6 +145,19 @@ def test_copy_feature_flag_update_existing(self):
             ensure_experience_continuity=False,
         )
 
+        # The following instances must remain linked to the existing flag after overwriting it
+        experiment = Experiment.objects.create(team=self.team_2, created_by=self.user, feature_flag_id=existing_flag.id)
```

Comment:
I thought of one other edge case for copying --->

This line https://github.com/PostHog/posthog/blob/master/posthog/api/organization_feature_flag.py#L96 isn't really correct, because:

1. A feature flag can be soft deleted
2. In this case, we _want_ to override everything, for all intents and purposes the flag is deleted for the user.

So, would be good to test for this case too, and update the line linked above to also filter for `deleted=False`

Reply:
yep indeed!